src/main/java/dk/kb/discover/webservice/OpenApiResource.java

```diff
@@ -0,0 +1,117 @@
+package dk.kb.discover.webservice;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dk.kb.util.Resolver;
+import dk.kb.util.yaml.YAML;
+import org.yaml.snakeyaml.Yaml;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class OpenApiResource {
+
+    public static final String APPLICATION_YAML = "application/yaml";
+
+    /**
+     * Pattern to allow search-replace for variabels defined as ${config.yaml.path} in OpenAPI specifications.
+     * Everything after 'config.' is treated as a path to an entry in the backing configuration.
```

Comment:
The extrapolation system in the `kb-util` `YAML` uses colon `:` as separator between the ID for the substitutor mechanism ant the key for the substitutor: `${config:yaml.path}`. To be consistent we should also use colon here.

Reply:
Yes, indeed!